code/game/jobs/job/civilian_chaplain.dm

```diff
@@ -2,135 +2,134 @@
 
 
 
-#/datum/job/chaplain
-#	title = "Chaplain"
-#	department = "Civilian"
-#	department_flag = CIV
-#
-#	total_positions = 1
-#	spawn_positions = 1
-#	supervisors = "the head of personnel"
-#	selection_color = "#515151"
-#	access = list(access_morgue, access_chapel_office, access_crematorium, access_maint_tunnels)
-#	minimal_access = list(access_morgue, access_chapel_office, access_crematorium)
-#	alt_titles = list("Counselor")
-#	outfit_type = /decl/hierarchy/outfit/job/chaplain
-#
-#	equip(var/mob/living/carbon/human/H, var/alt_title, var/ask_questions = TRUE)
-#		. = ..()
-#		if(!.)
-#			return
-#		if(!ask_questions)
-#			return
-#
-#		var/obj/item/weapon/storage/bible/B = locate(/obj/item/weapon/storage/bible) in H
-#		if(!B)
-#			return
-#
-#		spawn(0)
-#			var/religion_name = "Christianity"
-#			var/new_religion = sanitize(input(H, "You are the crew services officer. Would you like to change your religion? Default is Christianity, in SPACE.", "Name change", religion_name), MAX_NAME_LEN)
-#
-#			if (!new_religion)
-#				new_religion = religion_name
-#			switch(lowertext(new_religion))
```

Comment:
As far as I understand, the dead space setting has a religion called Unitology ?
It would probably be a good idea to include that here, perhaps even making it the only option.

Reply:
That was my thoughts too, so the chaplain in the dead space version becomes the Unitologist preacher, but for now just uncommenting this to resolve errors.